app/components/SettingsCellComponent.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  Dimensions,
+  Switch,
+} from 'react-native'
+import { colors } from '../config'
+
+const { settingsCellTextColor, settingsCellBorderLineColor } = colors
+const { width: screenWidth } = Dimensions.get('screen')
+
+const SettingsCellComponent = ({
+  cellText,
+  onCellPress,
+  onSwitchValueChange,
+  isSwitchCellComponent,
+}) => {
+  const cellPressed = () => {
+    if (onCellPress) {
+      onCellPress()
+    }
+  }
+
+  const switchValueChanged = () => {
+    if (onSwitchValueChange) {
+      onSwitchValueChange()
+    }
+  }
```

Comment:
Again can make this more concise:

```suggestion
const switchValueChanged = onSwitchValueChanged != null onSwitchValueChanged : null
```

Reply:
@MDHayes yes this can be concise